test/pkg/manifest/manifest.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+    <description>containerd-shim-runhcs-v1</description>
```

Comment:
>containerd-shim-runhcs-v1 [](http://example.com/codeflow?start=17&length=25)

update description?

Reply:
i totally missed that when i made the original PR
since its also in main, id rather get in a PR to main first that updates the description, then backport that to 0.9 and 0.11, if thats allright

Reply:
(#1919)

Reply:
Yeah back porting makes more sense